src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -1091,8 +1091,7 @@ def iot_central_app_list(client, resource_group_name=None):
 
 
 def iot_central_app_update(client, app_name, parameters, resource_group_name):
-    etag = parameters.additional_properties['etag']
-    return client.apps.update(resource_group_name, app_name, parameters, {'IF-MATCH': etag})
+    return client.apps.update(resource_group_name, app_name, parameters)
```

Comment:
May I ask why we don't need `etag` anymore?

Reply:
We got S360 issue warning us that swagger doesn't match with our RP response. We then removed the etag from our RP. After through testing, we realized that it will potentially break our az cli iot central app update command if the changes get out to prod. In order for RP changes to get out, we need az cli changes (without checking the etag) out first so that az cli iot central update command will continue to work. That is why we remove it now.

Reply:
we also have etag in our header just in case we needed it.

Reply:
Got it, thanks~